src/machine.rs

```diff
@@ -341,11 +341,14 @@ impl<'m> Machine<'m> {
         }
 
         trace!("{vm_id}: Deleting VM resources...");
+        let vm_dir = jailer_workspace_dir
```

Comment:
maybe we can add a comment explaining that we are adding "root" just like [here](https://github.com/blockjoy/firec/blob/main/src/config/jailer.rs#L196)? otherwise it might look scary to remove the parent dir